doc/examples/applications/plot_solidification_tracking.py

```diff
@@ -0,0 +1,147 @@
+)
+plotly.io.show(fig)
+
+#####################################################################
+# Compute image deltas
+# ====================
+# Let us apply a Gaussian low-pass filter to the images in order to smooth
+# the images and reduce noise.
+# Next, we compute the image deltas, i.e., the sequence of differences
+# between two consecutive frames. To do this, we subtract ``image_sequence``
+# from itself, but offset by one frame so that the subtracted images are 
+# one frame behind in time.
+
+images_smoothed = filters.gaussian(image_sequence)
+image_deltas = images_smoothed[1:, :, :] - images_smoothed[:-1, :, :]
+
+fig = px.imshow(
+    image_deltas,
+    animation_frame=0,
+    binary_string=True,
+    labels={'animation_frame': 'time point'}
+)
+plotly.io.show(fig)
+
+#####################################################################
+# Clip lowest and highest intensities
+# ===================================
+# We now calculate the 5th and 95th percentile intensities of ``image_deltas``:
+# we want to clip the intensities which lie below the 5th percentile
+# intensity and above the 95th percentile intensity, while also rescaling 
+# the intensity values to [0, 1]. 
+
+p_low, p_high = np.percentile(image_deltas, [5, 95])
+clipped = image_deltas - p_low
+clipped[clipped < 0.0] = 0.0
+clipped = clipped / p_high
+clipped[clipped > 1.0] = 1.0
+
+fig = px.imshow(
+    clipped,
+    animation_frame=0,
+    binary_string=True,
+    labels={'animation_frame': 'time point'}
+)
+plotly.io.show(fig)
+
+#####################################################################
+# Invert and denoise images
+# =========================
+# Next, we'll invert the images so the regions of highest intensity 
+# will correspond to the region we're interested in tracking (i.e. the 
+# solid-liquid interface). With the images inverted, we'll apply a total 
+# variation denoising filter te reduce some of the noise beyond the interface.
+
+inverted = 1 - clipped
+denoised = restoration.denoise_tv_chambolle(inverted, weight=0.15)
+
+fig = px.imshow(
+    denoised,
+    animation_frame=0,
+    binary_string=True,
+    labels={'animation_frame': 'time point'}
+)
+plotly.io.show(fig) 
+
+#####################################################################
+# Create binary images
+# ====================
+# Our next step is to create binarize the images, splitting each image 
+# into a foreground and a background. We want the solid-liquid interfaces 
+# to be the most prominent features in the foreground of these binary images 
+# so that the interfaces can eventually be separated from the rest of the 
+# image. First, we create an empty image (NumPy array full of zeros) matching
+# the size of the images processed so far:
+
+print(f'{denoised.shape=}')
+mask = np.zeros_like(denoised)
+print(f'{mask.shape=}')
+
+#####################################################################
+# With an empty array the same size of our images, we now set 
```

Comment:
```suggestion
# so that it can eventually be separated from the rest of the image.
# 
# We need 
```
(creating the mask does not deserve further development)